src/appkit.cs

```diff
@@ -27689,4 +27861,148 @@ interface NSSharingServicePickerToolbarItemDelegate
 		[Export ("itemsForSharingServicePickerToolbarItem:")]
 		NSObject[] GetItems (NSSharingServicePickerToolbarItem pickerToolbarItem);
 	}
+
+	[Unavailable (PlatformName.MacCatalyst)]
+	[Mac (10,16)]
+	[BaseType (typeof(NSToolbarItem))]
+	interface NSSearchToolbarItem
+	{
+		[DesignatedInitializer]
+		[Export ("initWithItemIdentifier:")]
+		IntPtr Constructor (string itemIdentifier);
+
+		[Export ("searchField", ArgumentSemantic.Strong)]
+		NSSearchField SearchField { get; set; }
+
+		[Export ("resignsFirstResponderWithCancel")]
+		bool ResignsFirstResponderWithCancel { get; set; }
+
+		[Export ("minimumWidthForSearchFieldRepresentation")]
+		nfloat MinimumWidthForSearchFieldRepresentation { get; set; }
+
+		[Export ("preferredWidthForSearchFieldRepresentation")]
+		nfloat PreferredWidthForSearchFieldRepresentation { get; set; }
+	}
+
+	delegate NSView NSTableViewDiffableDataSourceCellProvider (NSTableView tableView, NSTableColumn column, nint row, NSObject itemId);
+	delegate NSTableRowView NSTableViewDiffableDataSourceRowProvider (NSTableView tableView, nint row, NSObject identifier);
+	delegate NSView NSTableViewDiffableDataSourceSectionHeaderViewProvider (NSTableView tableView, nint row, NSObject sectionId);
+
+	[Mac (10,16)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface NSTableViewDiffableDataSource : NSTableViewDataSource
```

Comment:
that does not match the native side
```
@interface NSTableViewDiffableDataSource<SectionIdentifierType,ItemIdentifierType> : NSObject<NSTableViewDataSource>
```
and that's a similar issue to what we had in Foundation (and UIKit) that requires generator changes (iirc from last year)